### PR TITLE
remove excluded postgres native types

### PR DESCRIPTION
**Pull request by carmenberndt**
closes https://github.com/prisma/prisma-engines/issues/1323

### CODE REVIEW

File: introspection-engine/introspection-engine-tests/tests/tables/mod.rs [hunk 1]
```diff
@@ -4,15 +4,21 @@ use introspection_engine_tests::{assert_eq_datamodels, test_api::*};
 use quaint::prelude::Queryable;
 use test_macros::test_each_connector_mssql as test_each_connector;
 
-#[test_each_connector]
+#[test_each_connector(tags("postgres", "mysql", "sqlite"))]
```
Comment (pimeys) [inline introspection-engine/introspection-engine-tests/tests/tables/mod.rs:7]: Not testing MSSQL here?

**Review by pimeys — CHANGES_REQUESTED** [review]

File: introspection-engine/introspection-engine-tests/tests/tables/mod.rs [hunk 2]
```diff
@@ -4,15 +4,21 @@ use introspection_engine_tests::{assert_eq_datamodels, test_api::*};
...
     api.barrel()
         .execute_with_schema(
             |migration| {
-                migration.create_table("Blog", |t| {
+                migration.create_table("Blog", move |t| {
                     t.add_column("bool", types::boolean());
                     t.add_column("float", types::float());
-                    t.add_column("date", types::datetime());
+                    if sql_family.is_postgres() {
+                        t.inject_custom("date Timestamp Not Null");
```
Comment (pimeys) [inline introspection-engine/introspection-engine-tests/tests/tables/mod.rs:18]:
```
t.add_column("date", types::timestamp().nullable(false))
```

File: introspection-engine/introspection-engine-tests/tests/tables/mod.rs (hunk 1 above)
Comment (do4gr) [reply introspection-engine/introspection-engine-tests/tests/tables/mod.rs:7]: Ah, good spot, I forgot to revert that change

File: introspection-engine/introspection-engine-tests/tests/tables/mod.rs (hunk 2 above)
Comment (pimeys) [reply introspection-engine/introspection-engine-tests/tests/tables/mod.rs:18]: If missing that function, you can add it to this branch: https://github.com/prisma/barrel/tree/mssql-support

File: introspection-engine/introspection-engine-tests/tests/tables/mod.rs [hunk 3]
```diff
@@ -299,10 +307,15 @@ async fn default_values(api: &TestApi) -> crate::TestResult {
                     t.add_column("int_static", types::integer().default(2).nullable(true));
                     t.add_column("float_static", types::float().default(1.43).nullable(true));
                     t.add_column("boolean_static", types::boolean().default(true).nullable(true));
-                    t.add_column(
-                        "datetime_now",
-                        types::datetime().default(functions::current_timestamp()).nullable(true),
-                    );
+
+                    if sql_family.is_postgres() {
+                        t.inject_custom("datetime_now Timestamp Default current_timestamp");
```
Comment (pimeys) [inline introspection-engine/introspection-engine-tests/tests/tables/mod.rs:312]:
```rust
t.add_column("datetime_now"), types::timestamp().default(functions::current_timestamp()))
```

File: introspection-engine/introspection-engine-tests/tests/tables/mod.rs (hunk 2 above)
Comment (do4gr) [reply introspection-engine/introspection-engine-tests/tests/tables/mod.rs:18]: There is no timestamp type, but time works as well for the purposes of this test.

File: introspection-engine/introspection-engine-tests/tests/tables/mod.rs (hunk 3 above)
Comment (do4gr) [reply introspection-engine/introspection-engine-tests/tests/tables/mod.rs:312]: There is no timestamp type, but time works as well for the purposes of this test.

Comment (pimeys) [reply introspection-engine/introspection-engine-tests/tests/tables/mod.rs]: We could maybe change this to be `timestamp` here, then we wouldn't need this `if`: https://github.com/prisma/barrel/blob/mssql-support/src/backend/pg.rs#L219

**Review by pimeys — APPROVED** [review]

**Pull request merged**